suse_caasp

```diff
@@ -378,20 +382,218 @@ if check_rpm kubernetes-client && [ -f /etc/kubernetes/admin.conf ]; then
     log_cmd $OF 'kubectl version'
     log_cmd $OF 'kubectl api-versions'
     log_cmd $OF 'kubectl config view'
+    log_cmd $OF 'kubectl -n kube-system get pods --field-selector=status.phase=Running'
+    log_cmd $OF 'kubectl get events --sort-by=.metadata.creationTimestamp'
     log_cmd $OF 'kubectl get nodes'
+    log_cmd $OF 'kubectl get all -A'
+    log_cmd $OF 'kubectl get nodes -o yaml'
```

Comment:
I think it's safe if we also do a `kubectl get all -n kube-system -o yaml`, we would have many information about the cluster.

Reply:
It provides more information than a plain table.